flixel/animation/FlxAnimation.hx

```diff
@@ -277,4 +277,20 @@ class FlxAnimation extends FlxBaseAnimation
 	{
 		return _frames.length;
 	}
+	
+	/**
+	 * Appends the reverse of the animation to its frames, without duplicating the original last frame.
+	 * 
+	 * If looped is true, the original first frame is also not duplicated.
+	 * 
+	 * E.g. [0,1,2,3] becomes [0,1,2,3,2,1]   (looped)     -- Frame 0 is not played twice in a row when the loop returns to the beginning.
+	 * but  [0,1,2,3] becomes [0,1,2,3,2,1,0] (non-looped) -- Animation ends looking how it started.
+	 */
+	public function pingPongifyFrames()
```

Comment:
I'm open to name suggestions.  :)

Didn't see #1888.  That would help, then my function could just live in my own code.  But I thought it would be a nice helper function to have, since I can picture many animations that one might want to play this way.  E.g. a bouncing ball, a person jumping, etc.  It is a cheap technique, but that's the point: you only need half the number of rendered frames this way, therefore half the memory usage.  I figured that would be a common enough scenario/technique that it was worth including in the core.

The method you indicate here would duplicate both the starting and ending frame, adding two extra pauses in the animation if you're looping it, and one extra pause in the middle if not.  Hence my function taking away 2 or 1 from the number of frames depending on `looped`, in line 292.